src/pages/en/guides/deploy/edgio.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Deploy your Astro Site to Edgio
+description: How to deploy your Astro site to the web using Edgio.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can deploy your Astro project to [Edgio](https://www.edg.io/), an edge and CDN platform to deploy, protect and accelerate websites and APIs.
+
+:::tip
+Check out [the Astro guide in Edgio’s docs](https://docs.edg.io/guides/astro)!
+:::
+
+## Install Edgio CLI
```

Comment:
Could we rework this to be a little more goal-oriented? The old Layer0 doc is a good example in the sense that the heading describes the task someone wants to do: “How to deploy” (could be a variant on this “Deploy your site to Edgio”, “How to deploy with Edgio”, etc.)

Then instead of having “Install Edgio CLI”, “Add Edgio to your Astro site” etc. as headings they are numbered steps for the things you need to do to accomplish the task.